<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menú de Operaciones</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../logo.ico</normaloff>../../../logo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelOrigen">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fondo (Origen)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbOrigen">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tfTitular">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Nombre y Apellidos</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOrigen_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Titular</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOrigen_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nº Participaciones</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tfParticipaciones">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelOrigen_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Importe</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tfImporte">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelOrigen_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string> -$/€-</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDestino">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fondo (Destino)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbDestino">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Ninguno</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelOrigen_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Valor (Origen)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tfValorOrigen">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>D+4 ..</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOrigen_7">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Valor (Destino)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxDestino">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="tfValorDestino">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>D+1 ...</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOrigen_8">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hora (Corte)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fHoraCorte">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonEnviar">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>340</y>
      <width>151</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Enviar Operación</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonBorrar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Borrar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBorrar</sender>
   <signal>clicked()</signal>
   <receiver>tfParticipaciones</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBorrar</sender>
   <signal>clicked()</signal>
   <receiver>tfImporte</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBorrar</sender>
   <signal>clicked()</signal>
   <receiver>tfValorOrigen</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBorrar</sender>
   <signal>clicked()</signal>
   <receiver>tfTitular</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBorrar</sender>
   <signal>clicked()</signal>
   <receiver>fHoraCorte</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBorrar</sender>
   <signal>clicked()</signal>
   <receiver>tfValorDestino</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
